### DISCUSSION TITLE
Raw markdown on Web

### DISCUSSION THREAD

**Discussion by cip123** [200ok-ch/organice#967] [General]
When on the Web version, I would like to be able to edit the file as in emacs using markdown.  

Basically, I would like to act as a text editor.

The only thing I would change is to use the mouse to click checkboxes, TODO states, etc, instead of Emacs shortcuts.

Is this something I can do with Organice?

Thank you.

**Comment by schoettl** [comment]
I don't quite understand. Organice is not a Markdown but an orgmode UI. Markdown support will not be added.

See also [here for how we work with issues](https://github.com/200ok-ch/organice/blob/master/CONTRIBUTING.org#how-we-work-with-issues).

**Comment by cip123** [comment]
OK, I was hoping it would be more desktop-friendly.
